sdk/storage/pom.service.xml

```diff
@@ -14,6 +14,8 @@
     <module>../core/azure-core-test</module>
     <module>../core/azure-core-http-netty</module>
     <module>../identity/azure-identity</module>
+    <module>../keyvault/microsoft-azure-keyvault-core</module>
```

Comment:
Should we be taking a dependency on track 1 Key Vault?

@JonathanGiles

Reply:
This dependency is purely for testing purposes so we can make sure track 1 and track 2 SDKs encryption is cross compatible.

Reply:
This is included in pom.service.xml as one of the sources to build. So, our pipelines will build track 1 and track 2 libraries.

Reply:
just removed from pom.service.xml